common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +34,13 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            application_log.exception(
+                "UnicodeDecodeError Event_type: %r, Event_source: %r, Page: %r, Referer: %r",
+                event.get('event_type'), event.get('event_source'), event.get('page'), event.get('referer')
+            )
```

Comment:
yes @adampalay good point here is the pr link for this https://github.com/edx/edx-platform/pull/10793